### PR TITLE
Adds an example to plot stages of tractometry.

### PR DESCRIPTION
**Pull request by arokem**
More visualizations and animations. Supersedes #1016

### CODE REVIEW

**Comment by 36000** [comment]
Maybe fury cannot be run in CI? Do we just want to add this to the list of exemptions?

**Comment by arokem** [comment]
Oh - good catch! I need to set up xvfb for this example, like I did in the other visualization example.

**Comment by 36000** [comment]
@arokem is this ready to merge?

**Comment by arokem** [comment]
Thanks for sorting out the CI! I do want to add more text here before merging, though. But probably not today...

File: .github/workflows/docbuild.yml [hunk 1]
```diff
@@ -21,11 +21,15 @@ jobs:
       run: |
         python -m pip install --upgrade pip
         pip install .[all]
-        conda install -c conda-forge xvfbwrapper
+        sudo apt-get install -y xvfb
+    - name: Run XVFB
+      run: Xvfb :99 -screen 0 1280x1024x24
+      background: true
```
Comment (arokem) [inline .github/workflows/docbuild.yml:27]: Did this ever work @36000? I am getting this error: https://github.com/yeatmanlab/pyAFQ/actions/runs/7734636723

**Comment by 36000** [comment]
@arokem  is this ready to merge?

**Comment by arokem** [comment]
Give me 10 minutes to fix up the text...

**Comment by arokem** [comment]
OK - rebased and updated the text to be more explanatory of the different stages implemented here.

**Comment by arokem** [comment]
BTW: the CI failed with the recent merge of #1116, but I think that was just a connectivity glitch, so I wouldn't worry about it.

**Comment by arokem** [comment]
But this is a problem: https://github.com/yeatmanlab/pyAFQ/actions/runs/8269899416

File: .github/workflows/docbuild.yml [hunk 2]
```diff
@@ -23,8 +23,7 @@ jobs:
         pip install .[all]
         sudo apt-get install -y xvfb
     - name: Run XVFB
-      run: Xvfb :99 -screen 0 1280x1024x24
```
Comment (36000) [inline .github/workflows/docbuild.yml]: good idea

**Comment by arokem** [comment]
Looks like maybe all the CI passed with [cf3250b](https://github.com/yeatmanlab/pyAFQ/pull/1073/commits/cf3250b44022427f493a7d0a5729fea694826b69), so I am going to rebase back that and force push, so the last few commits will disappear.

**Comment by arokem** [comment]
I see what happened - it only _looks_ like [cf3250b](https://github.com/yeatmanlab/pyAFQ/pull/1073/commits/cf3250b44022427f493a7d0a5729fea694826b69) passed, but that's because the documentation jobs failed to launch.

**Comment by arokem** [comment]
If this doesn't pass now, I think that we'd be better off adding this example to the skip list, and moving on with a release.

**Comment by arokem** [comment]
Looks like it passed now! Or did I miss something? @36000 : when you get a chance, could you please take a look and merge if this looks good? I think that we can put out a release once this is in.

**Comment by 36000** [comment]
Yes, good job! Very finicky

**Pull request merged**